dashboard/src/utility/componentClassNames.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import classNames from 'classnames'
+
+const componentClassNames = (owner, ...args) => {
+  if (!React.Component.prototype.isPrototypeOf(owner))
```

Comment:
Super nit: I would sleep better if we put braces around all multi-line if-blocks.